src/NumericalAlgorithms/Interpolation/InterpolationTargetApparentHorizon.hpp

```diff
@@ -167,9 +178,18 @@ struct ApparentHorizon {
     // Put Strahlkorper and its ComputeItems, FastFlow,
     // and verbosity into a new DataBox.
     // Put the initial guess also into PreviousStrahlkorper.
-    Initialization::mutate_assign<simple_tags>(
-        box, options.initial_guess, options.fast_flow, options.verbosity,
-        options.initial_guess);
+    if constexpr (std::is_same_v<Frame, ::Frame::Inertial>) {
+      Initialization::mutate_assign<simple_tags>(
+          box, options.initial_guess, options.fast_flow, options.verbosity,
```

Comment:
this depends upon `StrahlkoperTags::items_tags<Frame>` being exactly `tmpl::list<Strahlkorper<Frame>>` so why not just use that in the `simple_tags` list above?